components/identity-core/org.wso2.carbon.identity.core/src/test/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilderTest.java

```diff
@@ -311,20 +311,20 @@ public void testGetAbsoluteURL(String protocol, String hostName, int port, Strin
                                 "fragment").addFragmentParameter("key2", "fragment").addFragmentParameter("key3",
                                 "fragment").addFragmentParameter("key4", "fragment").addParameter("key1", "v")
                                 .addParameter("key2", "v").addParameter("key3", "v").addParameter("key4", "v").build()
-                                .getAbsoluteInternalURL();
+                                .getAbsolutePublicURL();
```

Comment:
Shall we add a separate test case to cover getAbsoluteInternalURL as well?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2949/commits/2b1d2672a9098a2553b55ed732619275bc23ae27